public/js/app/pages/CreateNewMemo.jsx

```diff
@@ -377,18 +395,21 @@ function CreateNewMemo(props) {
                         data-testid="form-choose-new"
                         className="form-check"
                         id="choose-from-rounds-list"
-                        key={'new' + round.ladokRoundId}
+                        key={'new' + round.applicationCodes[0]}
                       >
                         <Input
                           type="checkbox"
                           data-testid="checkbox-choose-available-round"
-                          id={'new' + round.ladokRoundId}
+                          id={'new' + round.applicationCodes[0]}
                           name="chooseNew"
-                          value={round.ladokRoundId}
+                          value={round.applicationCodes[0]}
```

Comment:
`round.applicationCodes[0]` should be first checked if any application code exist in array, everywhere in the code

Reply:
Changed array to direct string. Now round.applicationCode will be used.